region/brinstar/kraid/Kraid Eye Door Room.json

```diff
@@ -129,6 +129,7 @@
         {"or": [
           "Morph",
           {"and": [
+            "canTrickyEnemyExtendRunway",
```

Comment:
Door 2 too

Reply:
I was only including it on door 1/3 when you need to run in the spikes to align the Zeb (no Morph). The top right door seemed much less tricky to me - I also didnt include it using a Zeb on the right door in Big Pink.